src/components/contributors.tsx

```diff
@@ -19,6 +19,10 @@ const contributors: GitHubProfileProps[] = [
         userName: 'EJAyarma',
         role: 'Front-End Engineer',
     },
+    {
+        userName: 'esmond-adjei',
+        role: 'mentee',
+    },
```

Comment:
Please update your role based on the assignment in the Scoping document in confluence